src/settings/backend.ts

```diff
@@ -12,8 +12,10 @@ const heldernetSettings: BackendSettings = {
   nodeUrls: ["https://lcd.heldernet.cosmwasm.com"],
 };
 
+const localPort = process.env.REACT_APP_STARGATE ? "26659" : "1317";
+
 const devnetSettings: BackendSettings = {
-  nodeUrls: ["http://localhost:1317"],
+  nodeUrls: [`http://localhost:${localPort}`],
```

Comment:
I'd split this in two different backends, once we have the stargate flag in `BackendSettings`